rotkehlchen/rotkehlchen.py

```diff
@@ -509,6 +510,8 @@ def unlock_user(
             method=self.data_updater.check_for_updates,
         )
 
+        self.addressbook_prioritizer = NamePrioritizer(self.data.db)
+
```

Comment:
You need to explain why we keep this in memory here. Also remove empty newline

```suggestion
        self.addressbook_prioritizer = NamePrioritizer(self.data.db)  # initialize here since it's reused by the api for addressbook endpoints
```